lib/requests.js

```diff
@@ -216,16 +216,21 @@ class PrepareRequest extends Request {
   }
 }
 class StartupRequest extends Request {
-  constructor(cqlVersion) {
+  constructor(cqlVersion, noCompact) {
     super();
     this.cqlVersion = cqlVersion || '3.0.0';
+    this.noCompact = noCompact || null;
```

Comment:
nit: `|| null` portion can be removed.
If `this.noCompact` is `truthy`, you insert `'NO_COMPACT': 'true'` which will work either way as you don't use the actual value.